Sources/PackageDescription/Target.swift

```diff
@@ -127,10 +127,12 @@ public final class Target {
     private var _pluginCapability: PluginCapability?
     
     /// The different types of capability that a plugin can provide. In this
-    /// version of SwiftPM, only build tool plugins are supported; this enum
-    /// will be extended as new plugin capabilities are added.
+    /// version of SwiftPM, only build tool and command plugins are supported;
+    /// this enum will be extended as new plugin capabilities are added.
     public enum PluginCapability {
         case _buildTool
+        @available(_PackageDescription, introduced: 999.0)
```

Comment:
should this be 5.6?

Reply:
I had it that way initially, but since it's still an experimental feature it seemed more proper to make it 999 here and then change it to 5.6 when the feature flag is removed after acceptance.  That way packages have to specifically opt into the mainline APIs via their toolsversion.